README.md

```diff
@@ -36,6 +32,25 @@ npm run dev
 ```
 Three test user URLs are printed on the command line that can be used to test the server.
 
+## Using Redis queue option
+By default the waiting room uses a local queue. To switch to Redis, first:
+Change line of code in server.js
```

Comment:
I think it would be better to configure this through a CLI argument instead of requiring users to change the code.

Reply:
Moved the redis stuff to a dev readme. will create an issue to look back into using redis